recipes/dgl/build.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -euxo pipefail
+
+rm -rf build || true
+
+CUDA_CMAKE_OPTIONS="-DCMAKE_CUDA_COMPILER=${CUDA_HOME}/bin/nvcc -DCMAKE_CUDA_HOST_COMPILER=${CXX}"
+
+# Remove -std=c++17 from CXXFLAGS for compatibility with nvcc
+export CXXFLAGS="$(echo $CXXFLAGS | sed -e 's/ -std=[^ ]*//')"
+
```

Comment:
```suggestion
export CFLAGS="$(echo $CFLAGS | sed -e 's/ -mtune=[^ ]*//')"
```

When compiling without cuda on GCC 12 I got error from third_party library (tensorpipe) - removing mtune flag fixed my issue